breeze

```diff
@@ -2800,12 +2800,12 @@ function breeze::check_and_save_all_params() {
 
     if [[ ${PYTHON_MAJOR_MINOR_VERSION} == "2.7" || ${PYTHON_MAJOR_MINOR_VERSION} == "3.5" ]]; then
         if [[ ${BRANCH_NAME} == "master" ]]; then
-            echo >&2
-            echo >&2 "The ${PYTHON_MAJOR_MINOR_VERSION} can only be used when installing Airflow 1.10.*"
-            echo >&2
-            echo >&2 "You can use it only when you specify 1.10 Airflow via --install-airflow-version"
-            echo >&2 "or --install-airflow-reference and they point to 1.10 version of Airflow"
-            echo >&2
+            echo
+            echo -e " \e[31mERROR: The ${PYTHON_MAJOR_MINOR_VERSION} can only be used when installing Airflow 1.10.*\e[0m"
```

Comment:
I think something like this would ease readability:

```suggestion
            echo " ${COLOR_RED}ERROR: The ${PYTHON_MAJOR_MINOR_VERSION} can only be used when installing Airflow 1.10.*${COLOR_RESET}"
```

```sh
COLOR_RED=$'\e[31m'
COLOR_RESET=$'\e[0m'
# etc.
```

(Or we could try associative arrays and then `${color[red]}`)

Reply:
Yeah. Why not :).

Reply:
I mean _I_ know that `31m` is red, but don't ask me about any other colours

Reply:
I prefer not to use arrays if not absolutely needed though - especially on Mac OS /Bash 3 they get tricky. And extracting the common prefixes (ERROR/OK/WARNING) in those vars is even better.

Reply:
> I mean _I_ know that `31m` is red, but don't ask me about any other colours

Well. I would not recognize CYAN or MAGENTA either ;)